packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -21,6 +21,7 @@
 #include <UI.Xaml.Controls.h>
 #include <UI.Xaml.Hosting.h>
 #include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.UI.Xaml.Automation.h>
```

Comment:
```suggestion
#include <UI.Xaml.Automation.h>
```

Reply:
OT, I wonder if we can create poisoned headers for WUX to tell folks to use the straddle types as a built-time error.

Reply:
that'd be nice. I'm not sure how since the straddle headers need to include the wux/mux headers eventually, and cppwinrt adds the include path

Reply:
A strategy you could use would be to disable poisoning via preprocessor definition.

E.g. have the poisoned WUX header wrap the real include in something like "#ifndef ALLOW_WUX". That lets you set and pop some state around the include, only in the straddle headers.

We can define the order of include path I think. We just need the poisoned header location to be searched first.

Reply:
https://github.com/microsoft/react-native-windows/issues/8255